src/tests/ftest/datamover/posix_preserve_props.py

```diff
@@ -0,0 +1,240 @@
+
+        Args:
+            cont (TestContainer): the container to verify.
+            prop_list (list): list of properties from get_cont_prop.
+
+        """
+        actual_list = self.get_cont_prop(cont)
+
+        # Make sure sizes match
+        if len(prop_list) != len(actual_list):
+            self.log.info("Expected\n%s\nbut got\n%s\n",
+                          prop_list, actual_list)
+            self.fail("Container property verification failed.")
+
+        # Make sure each property matches
+        for prop_idx, prop in enumerate(prop_list):
+            # This one is not set
+            if (api == "DFS") and ("OID" in prop_list[prop_idx] or "ROOTS" in prop_list[prop_idx]):
+                continue
+            if prop != actual_list[prop_idx]:
+                self.log.info("Expected\n%s\nbut got\n%s\n",
+                              prop_list, actual_list)
+                self.fail("Container property verification failed.")
+
+        self.log.info("Verified %d container properties:\n%s",
+                      len(actual_list), actual_list)
+
+    @staticmethod
+    def get_cont_usr_attr():
+        """Generate some user attributes"""
+        attrs = {}
+        attrs["attr1".encode("utf-8")] = "value 1".encode("utf-8")
+        attrs["attr2".encode("utf-8")] = "value 2".encode("utf-8")
+        return attrs
+
+    def verify_usr_attr(self, cont):
+        """Verify user attributes. Expects the container to be open.
+
+        Args:
+            cont (TestContainer): the container to verify.
+
+        """
+        attrs = self.get_cont_usr_attr()
+        actual_attrs = cont.container.get_attr(list(attrs.keys()))
+
+        # Make sure the sizes match
+        if len(attrs.keys()) != len(actual_attrs.keys()):
+            self.log.info("Expected\n%s\nbut got\n%s\n",
+                          attrs.items(), actual_attrs.items())
+            self.fail("Container user attributes verification failed.")
+
+        # Make sure each attr matches
+        for attr, val in attrs.items():
+            if attr not in actual_attrs:
+                self.log.info("Expected\n%s\nbut got\n%s\n",
+                              attrs.items(), actual_attrs.items())
+                self.fail("Container user attributes verification failed.")
+            if val != actual_attrs[attr]:
+                self.log.info("Expected\n%s\nbut got\n%s\n",
+                              attrs.items(), actual_attrs.items())
+                self.fail("Container user attributes verification failed.")
+        self.log.info("Verified %d user attributes:\n%s",
+                      len(attrs.keys()), attrs.items())
+
+    @avocado.fail_on(DaosApiError)
+    def test_dm_preserve_props_fs_copy_posix_dfs(self):
+        """
+        Test Description:
+            Verifies destination container creation
+            when API is unknown, including
+            container properties and user attributes.
+
+        :avocado: tags=all,full_regression
```

Comment:
can't we add this to daily regression at least?

Reply:
@mchaarawi https://daosio.atlassian.net/browse/DAOS-9001 I created a ticket I can do this as soon as this merges which should be fairly quick if I just update the test file.

Reply:
@mchaarawi closed that ticket and just added it to daily_regression since I had to update PR anyway.